tests/src/test/scala/ch/renku/acceptancetests/workflows/FlightsTutorial.scala

```diff
@@ -57,7 +73,7 @@ trait FlightsTutorial extends GivenWhenThen with Matchers with ScalatestMatchers
 
     And("runs step 1")
     console %> c"mkdir $projectFolder/data/output"
-    console %> c"renku run python src/00-FilterFlights.py data/201901_us_flights_1/2019-01-flights.csv.zip data/output/2019-01-flights-filtered.csv"
+    console %> c"./venv/bin/python -m renku.cli run ./venv/bin/python src/00-FilterFlights.py data/201901_us_flights_1/2019-01-flights.csv.zip data/output/2019-01-flights-filtered.csv"
```

Comment:
If you activate the virtualenv you won't have to have full paths here which will make this quite a bit easier to read and potentially avoid some pitfalls for others who might be adding steps in the future

Reply:
I agree that specifying the full path is fragile and prone to typing errors